test/ably/restpush_test.py

```diff
@@ -240,4 +241,50 @@ def test_admin_device_registrations_remove_where(self):
         with pytest.raises(AblyException): get(device_id) # Doesn't exist
 
         # Remove with no matching params
-        assert remove_where(clientId=data['clientId']).status_code == 204
+        assert remove_where(clientId=client_id).status_code == 204
+
+    # RSH1c3
+    def test_admin_channel_subscriptions_save(self):
+        save = self.ably.push.admin.channel_subscriptions.save
+
+        # Register device
+        device_id = self.get_device_id()
+        data = {
+            'id': device_id,
+            'platform': 'ios',
+            'formFactor': 'phone',
+            'push': {
+                'recipient': {
+                    'transportType': 'apns',
+                    'deviceToken': DEVICE_TOKEN
+                }
+            },
+        }
+        self.__save(data)
+
+        # Subscribe
+        channel = 'canpublish:test'
+        subscription = PushChannelSubscription(channel, device_id=device_id)
+        subscription = save(subscription)
+        assert type(subscription) is PushChannelSubscription
+        assert subscription.channel == channel
+        assert subscription.device_id == device_id
```

Comment:
Do you not think we should reload the the `subscription` to ensure what you are checking is not in fact the model stored in memory as opposed to retrieved from the REST API?

Reply:
I agree. Also, while you are at it, I'd test also other fields.
Finally, in future PRs, can we test also the 'android' platform?